ads/inabox/inabox-messaging-host.js

```diff
@@ -241,3 +252,34 @@ export class InaboxMessagingHost {
     return null;
   }
 }
+
+/**
+ * Returns true if win's properties can be accessed and win is defined.
+ * This functioned is used to determine if a window is cross-domained
+ * from the perspective of the current window.
+ * @param {Window} win
+ * @return {boolean}
+ */
+function canInspectWindow(win) {
+  try {
+    return !!win && win.location.href != null && canTouchProperty(win, 'test');
```

Comment:
I think this can just be the following.  If win or win location do not exist, it will throw which is equivalent:

try {
   const /* eslint no-unused-vars: 0 */ unused = win.location.href && win['test'];
   return true;
} catch (unusedErr) {
   return false;
}

Reply:
done